packages/frontend/components/product/card/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { ColorPalette } from '@vse-bude/shared';
+import { css } from '@emotion/react';
+import type { Theme } from '../../../theme';
+
+export const productFooter = () => css`
+  margin-top: 15px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
Please organize your CSS props in order: https://9elements.com/css-rule-order/. Add take margins and paddings from theme